fieldservice_sale/tests/test_fsm_sale_common.py

```diff
@@ -95,3 +95,13 @@ def setUpFSMProducts(cls):
             'field_service_tracking': 'line',
             'fsm_order_template_id': cls.fsm_template_4.id,
         })
+        cls.standard_product_1 = cls.env['product.product'].create({
```

Comment:
I noticed this new product has not been used in the tests.  Did you intend to use it or can it be removed?

Reply:
My intention was to use it in `sale_order_5` test, I have already corrected it, thanks!